examples/onoff_light_bt/src/dev_att.rs

```diff
@@ -0,0 +1,165 @@
+/*
+ *
+ *    Copyright (c) 2020-2022 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+use rs_matter::data_model::sdm::dev_att::{DataType, DevAttDataFetcher};
+use rs_matter::error::{Error, ErrorCode};
+
+pub struct HardCodedDevAtt {}
+
+impl HardCodedDevAtt {
+    pub const fn new() -> Self {
+        Self {}
+    }
+}
+
+// credentials/examples/ExamplePAI.cpp FFF1
+const PAI_CERT: [u8; 463] = [
+    0x30, 0x82, 0x01, 0xcb, 0x30, 0x82, 0x01, 0x71, 0xa0, 0x03, 0x02, 0x01, 0x02, 0x02, 0x08, 0x56,
```

Comment:
what format is this? Could we do `include_bytes!` for rust as this data is not actually reviewable as code?

Reply:
I've created [this issue ](https://github.com/project-chip/rs-matter/issues/196) about it and in general what to do about the examples. Reason being - let's keep the PRs focused.